### ISSUE TITLE
Project is not working proprely

### ISSUE THREAD

**Issue by CoderRathore** [sger/laravel-messenger-demo#1]
I have download the code from github. And Also run the command by cmd but project not working properly.when we click on the login menu this error is coming "No supported encrypter found. The cipher and / or key length are invalid." So please solve this issue .

**Comment by javaftper** [comment]
Have you tried running `php artisan key:generate` from the console?

**Comment by CoderRathore** [comment]
yes i was run this command in cmd get Application key [pefMyY4nR9SgXGgEwKvpVk026HAwCd8D] set successfully.
but show this error
No supported encrypter found. The cipher and / or key length are invalid. 
please help me to slove this problem

**Comment by CoderRathore** [comment]
hi,

Project is working properly not chaing functionlity is not working.

**Comment by bosz** [comment]
Hi. The project does not work. 
Running `composer update` or `php artisan migrate` or `php artisan key:generate`  or `composer dumpautoload` gives me this error 

```
PHP Fatal error:  Uncaught Error: Class 'Illuminate\Foundation\Application' not found in /var/www/html/test/laravel-messenger-demo/bootstrap/app.php:14
Stack trace:
#0 /var/www/html/test/laravel-messenger-demo/artisan(18): require_once()
#1 {main}
  thrown in /var/www/html/test/laravel-messenger-demo/bootstrap/app.php on line 14
```

**Help pls**

**Comment by sger** [comment]
@bosz update to the latest laravel version.

**Comment by zilveer** [comment]
Any updates on this?
